homeassistant/components/iotty/const.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for the iotty integration."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.const import Platform
+
+DOMAIN = "iotty"
+
+OAUTH2_AUTHORIZE = "https://auth.iotty.com/.auth/oauth2/login"
+OAUTH2_TOKEN = "https://auth.iotty.com/.auth/oauth2/token"
+OAUTH2_CLIENT_ID = "hass-iotty"
+
+IOTTYAPI_BASE = "https://homeassistant.iotty.com/"
```

Comment:
These are only used once, move them to `api.py` and `application_credentials.py` respectively please.

Reply:
Done